packages/loot-core/src/server/cloud-storage.ts

```diff
@@ -304,7 +304,9 @@ export async function upload() {
           ? { 'X-ACTUAL-ENCRYPT-META': JSON.stringify(uploadMeta) }
           : null),
         ...(groupId ? { 'X-ACTUAL-GROUP-ID': groupId } : null),
-      },
+        // TODO: fix me
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      } as any,
```

Comment:
Moving away from all the `.testing.ts`, `.web.ts`, etc. custom extensions exposes new issues. That's a good signal